libdevice/nativecpu_utils.cpp

```diff
@@ -103,7 +103,7 @@ DefSubgroupBlockINTEL(uint8_t) DefSubgroupBlockINTEL(uint16_t)
 #define DefineGOp1(spir_sfx, name)\
 DEVICE_EXTERN_C bool __mux_sub_group_##name##_i1(bool);\
 DEVICE_EXTERN_C bool __mux_work_group_##name##_i1(uint32_t id, bool val);\
```

Comment:
```suggestion
DEVICE_EXTERN_C bool __mux_work_group_##name##_i1(int32_t id, bool val);\
```
And likewise in other places below?

Reply:
does id represent SPIR-V enum token value? I don't see scope or group operation parameter is used in the call of __mux_work_group_* functions.

Reply:
Oops, quite right, in this instance it does not match one of the parameters for the corresponding `__spirv_` function so this change, if we want it at all, should still not be part of this PR. In that case, the NativeCPU part looks good to me, thanks.